rmgpy/data/kinetics/family.py

```diff
@@ -1877,7 +1877,10 @@ def getKineticsFromDepository(self, depository, reaction, template, degeneracy):
                 kineticsList.append([deepcopy(entry.data), entry, entry.item.isIsomorphic(reaction, eitherDirection=False)])
         for kinetics, entry, isForward in kineticsList:
             if kinetics is not None:
-                kinetics.comment += "Matched reaction {0} {1} in {2}".format(entry.index, entry.label, depository.label)
+                kinetics.comment += "Matched reaction {0} {1} in {2}\nThis reaction matched rate rule {3}".format(entry.index, 
```

Comment:
Why is this needed? When we match a training reaction exactly, we technically aren't matching a rate rule.

Also, assuming that this is added, does it break uncertainty?

Reply:
That's a really good comment (and made me think back a bit). Here's the logic in why I think it is necessary: When loading a reaction from a chemkin file, I think a desired output would be similar information to when the reaction was saved in a chemkin file. Loading reactions from chemkin files does not currently require a loaded database. If only the depository index was saved (as it is before this PR), it would not be possible to repopulate the TemplateReaction.template attribute for reaction objects. In my isotope work, I need the template attribute to be populated so I can ensure that I don't include different transition states when generating reactions with isotopes.

Reply:
For the potential to break uncertainty, I did a visual inspection of uncertainty parsing in `family.py` when first making this commit. To double check, I just added the extra line in kinetics which is tested for uncertainty by `TestKineticsParsing` in `kineticsTest.py`, which still passes with the added line.

Reply:
Awesome observation! Fortunately, this comment is used in other parts of the code, so "'This reaction matched rate rule'" is already on the list of line breaks in the master branch.

Reply:
Ah, I see.